### ISSUE TITLE
[BingAIClient issue] If the message type equals 'InternalLoaderMessage', some tokens will be missing.

### ISSUE THREAD

**Issue by somanchiu** [waylaidwanderer/node-chatgpt-api#527]
**Describe the bug**
If the message type equals 'InternalLoaderMessage', the constant value "difference" is incorrect. It will lost some tokens.

**To Reproduce**
1. Use BingAIClient and send the message "Tell me a long story.".

**Expected behavior**
All tokens will passed to onProgress.

**Screenshots**
![image](https://github.com/waylaidwanderer/node-chatgpt-api/assets/8416023/52670670-658a-48a3-a800-cd075ae7d737)

**Node.js version (please complete the following information):**
 - OS: Windows
 - Version: v20.11.0

**Package version (please complete the following information):**
 - 1.37.0

**Additional context**
N/A

**Comment by jegue** [comment]
. Is the Bing Jailbreak still working?

**Comment by somanchiu** [comment]
> . Is the Bing Jailbreak still working?

Yes. Check the pull request [481](https://github.com/waylaidwanderer/node-chatgpt-api/pull/481).

**Comment by jegue** [comment]
> > . Is the Bing Jailbreak still working?
> 
> Yes. Check the pull request [481](https://github.com/waylaidwanderer/node-chatgpt-api/pull/481).

I mean, now is https://copilot.microsoft.com/ not Bing Chatai anymore
